wordpress-mobile/WordPress-iOS
# [Stats Refresh] Reload Stats when application will enter foreground

Fixes #12823 

This PR improves the stats refresh when the application will enter in the foreground state. The Stats dashboard, Stats detail and Post detail will observe the application state and in case refresh the data updating also the current date (keeping the right format).

## To test:
- Run the branch and open Stats from a selected site (ex. en.blog)
- Put the app in background and then in foreground, checking the logs to see if actually the API are called again.
- Put the app in background, change your device date (adding one or 2 days) and bring the app in foreground. Stats should update to the current date.

Update release notes:
